frontend/components/site/siteBuildLogs.jsx

```diff
@@ -26,6 +52,11 @@ const SiteBuildLogs = () => {
           <li><DownloadBuildLogsButton buildId={buildId} buildLogsData={logs} /></li>
         </ul>
       </div>
+      {buildTooOld(buildDetails) && (
+      <div>
+        <SiteBuildLogTable buildLogs={['Builds more than 180 days old are deleted according to platform policy.']} />
+      </div>
+      )}
       {(!logs || logs?.length === 0) && (
       <div>
         <SiteBuildLogTable buildLogs={['This build does not have any build logs.']} />
```

Comment:
```suggestion
      {(!logs || logs?.length === 0) && (
        <div>
          {buildTooOld(buildDetails) && (
            <SiteBuildLogTable buildLogs={['Builds more than 180 days old are deleted according to platform policy.']} />
          )}
          {!buildTooOld(buildDetails) && (
            <SiteBuildLogTable buildLogs={['This build does not have any build logs.']} />
          )}
```

Reply:
It just needs a little more to work:
```suggestion
      {(!logs || logs?.length === 0) && (
        <div>
          {buildTooOld(buildDetails) && (
            <SiteBuildLogTable buildLogs={['Builds more than 180 days old are deleted according to platform policy.']} />
          )}
          {!buildTooOld(buildDetails) && (
            <SiteBuildLogTable buildLogs={['This build does not have any build logs.']} />
          )}
        </div>
      )}
```

Reply:
Oh, I may have just snipped that in a different place from you. Never mind. 

Here's the approach I came up with though:
```javascript
      {(() => {
        if (!logs || logs?.length === 0) {
          if (buildTooOld(buildDetails)) {
            return (
              <div>
                <SiteBuildLogTable buildLogs={['Builds more than 180 days old are deleted according to platform policy.']} />
              </div>
            );
          }
          return (
            <div>
              <SiteBuildLogTable buildLogs={['This build does not have any build logs.']} />
            </div>
          );
        }
        return null;
      })()}
```

Which do you think is more readable?

Reply:
oh i like this one just fine! you don't have to wrap the `<SiteBuildLogTable>` in a `<div>` if you don't need to for layout purposes. It's just that the `return ()` statement can only have one parent node, so if you have two elements, we wrap them. Each of these is in their own return, and only one element, so you could save yourself 4 lines if you like :)

Reply:
Good point, thanks!

Reply:
What about?

```jsx
      {buildTooOld(buildDetails) ? (
        <SiteBuildLogTable
          buildLogs={[
            'Builds more than 180 days old are deleted according to platform policy.',
          ]}
        />
      ) : (
        <>
          {logs && logs?.length > 0 ? (
            <SiteBuildLogTable buildLogs={logs} buildState={state} />
          ) : (
            <SiteBuildLogTable
              buildLogs={['This build does not have any build logs.']}
            />
          )}
        </>
      )}
```

Reply:
Do you prefer this to what I pushed?

Reply:
I was wondering if a ternary operator would 1) work, 2) be easy to read. I could see a case for either approach; they're basically indistinguishable. I think ternaries are cool, but they're sometimes hard to read. I'm neutral on this :)

Reply:
Giving this a gentle bump, @apburnes. I think I'd already pushed my approach to this when you suggested another way here, but I don't know whether you'd like me to switch to your version or were just in the process of suggesting it when I applied mine. 

tl/dr: Am I waiting for you to re-review or are you waiting for me to switch to your suggested implementation?